Makefile

```diff
@@ -125,10 +125,11 @@ lt-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/load_test | kubectl apply -f -
 
-.PHONY: local-deploy-with-watcher
-local-deploy-with-watcher: install manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+.PHONY: local-deploy-with-watcher-secured
```

Comment:
This is a left over, originally I planned to have both (secured and unsecured), but that would make the Docs way harder to grasp, and since the actual setup should be secured, I just kept that one. I will rename it!